working/augmentation-libraries/feature-specification.md

```diff
@@ -347,34 +347,27 @@ invoked with the same value for `this`.
 
 ### Augmenting types
 
-A class, mixin, enum, or extension declaration can be marked with an `augment`
-modifier:
+A class, enum, extension, extension type, or mixin declaration can be marked
+with an `augment` modifier:
 
 ```dart
 augment class SomeClass {
   // ...
 }
 ```
 
-This means that instead of creating a new type declaration, the augmentation
-modifies a corresponding declaration in the augmented library or one of its
-other augmentations.
+This means that instead of creating a new declaration, the augmentation modifies
+a corresponding declaration in the augmented library or one of its other
+augmentations.
 
-All the keywords (other than `augment`) must be identical between the original
-declaration and the augmentation. This is to ensure that looking at either one
-will give a complete depiction of the capabilities of the type, and an
-augmentation cannot introduce hidden restrictions.
```

Comment:
I removed this paragraph because the compile errors section below restates the same thing.